source/views/guides/group-by-and-queryable-by-name.html.slim

```diff
@@ -0,0 +1,118 @@
+section.banner
+  .content-wrapper
+    .banner-wrapper
+      h2.banner__heading Group By and QueryableByName
+
+main
+  section.demo
+    .content-wrapper
+      .guide-wrapper
+        markdown:
+          Diesel can't statically check all queries due to the limitless
+          combinations of SQL. When you can't represent your query with
+          Diesel's DSL, you can fallback to
+          [`sql_query`](http://docs.diesel.rs/diesel/fn.sql_query.html).
+          Aggregations such as `GROUP BY` provide a great learning example for
```

Comment:
Maybe add a quick link to a page describing `GROUP BY` or paraphrase how the aggregation transforms the data